django/contrib/staticfiles/storage.py

```diff
@@ -58,11 +58,11 @@ class HashedFilesMixin:
                 '//# sourceMappingURL=%(url)s',
             ),
             (
-                r"""(?P<matched>import\s+(?s:(?P<imports>.*?))\s*from\s*["'](?P<url>.*?)["'])""",
+                r"""(?P<matched>import\s+(?s:(?P<imports>[_$a-zA-Z0-9\*\s{},]*?))\s*from\s*["'](?P<url>.*?)["'])""",
                 'import %(imports)s from "%(url)s"',
             ),
             (
-                r"""(?P<matched>export\s+(?s:(?P<exports>.*?))\s*from\s*["'](?P<url>.*?)["'])""",
+                r"""(?P<matched>export\s+(?s:(?P<exports>[_$a-zA-Z0-9\*\s{},]*?))\s*from\s*["'](?P<url>.*?)["'])""",
```

Comment:
As far as I'm concerned special characters lose their special meaning inside sets:
```suggestion
                r"""(?P<matched>export\s+(?s:(?P<exports>[_$a-zA-Z0-9*\s{},]*?))\s*from\s*["'](?P<url>.*?)["'])""",
```

Reply:
Yes. I went back and forth on that. 👍